sqlglot/dialects/duckdb.py

```diff
@@ -219,7 +219,6 @@ class Parser(parser.Parser):
         FUNCTIONS = {
             **parser.Parser.FUNCTIONS,
             "ARRAY_HAS": exp.ArrayContains.from_arg_list,
-            "ARRAY_LENGTH": exp.ArraySize.from_arg_list,
```

Comment:
why is this gone?

Reply:
moved in `_sql_names` because it's common